conf/routes

```diff
@@ -113,3 +113,100 @@ DELETE /admin/v1/users          @org.sagebionetworks.bridge.play.controllers.Use
 # Admin - Cache Management
 GET    /admin/v1/cache           @org.sagebionetworks.bridge.play.controllers.CacheAdminController.listItems
 DELETE /admin/v1/cache/:cacheKey @org.sagebionetworks.bridge.play.controllers.CacheAdminController.removeItem(cacheKey: String)
+
+# v3 routes
+# -----------------------------------------------------------------------------------------------------------------------
+
+# Authentication
+POST   /v3/auth/signIn                  @org.sagebionetworks.bridge.play.controllers.AuthenticationController.signIn
+GET    /v3/auth/signOut                 @org.sagebionetworks.bridge.play.controllers.AuthenticationController.signOut
+POST   /v3/auth/requestResetPassword    @org.sagebionetworks.bridge.play.controllers.AuthenticationController.requestResetPassword
+POST   /v3/auth/resetPassword           @org.sagebionetworks.bridge.play.controllers.AuthenticationController.resetPassword
+POST   /v3/auth/signUp                  @org.sagebionetworks.bridge.play.controllers.AuthenticationController.signUp
+POST   /v3/auth/verifyEmail             @org.sagebionetworks.bridge.play.controllers.AuthenticationController.verifyEmail
+POST   /v3/auth/resendEmailVerification @org.sagebionetworks.bridge.play.controllers.AuthenticationController.resendEmailVerification
+
+# Users
+POST   /v3/users                  @org.sagebionetworks.bridge.play.controllers.UserManagementController.createUser
+POST   /v3/users/email            @org.sagebionetworks.bridge.play.controllers.StudyController.sendStudyParticipantsRoster
```

Comment:
I feel like POST /v3/users/email doesn't really describe what this API does. I think we should have a name with "sendParticipantRoster" somewhere in there.